app/models/generic_object.rb

```diff
@@ -17,6 +17,9 @@ class GenericObject < ApplicationRecord
            :custom_actions, :custom_action_buttons,
            :to => :generic_object_definition, :allow_nil => true
 
+  delegate :name, :to => :generic_object_definition,       :prefix => true, :allow_nil => false
```

Comment:
Can we remove the extra spaces in the line?

Reply:
sure

Reply:
Done in the last update.